flink-end-to-end-tests/flink-end-to-end-tests-common-elasticsearch/pom.xml

```diff
@@ -43,50 +43,52 @@ under the License.
 		</dependency>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-connector-elasticsearch6</artifactId>
+			<artifactId>flink-connector-test-utils</artifactId>
 			<version>${project.version}</version>
+			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-elasticsearch-base</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>elasticsearch</artifactId>
+			<version>${testcontainers.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-end-to-end-tests-common</artifactId>
+			<version>${project.version}</version>
+			<scope>test</scope></dependency>
 	</dependencies>
 
+	<dependencyManagement>
+	   <dependencies>
+		  <dependency>
+			 <groupId>org.apache.httpcomponents</groupId>
+			 <artifactId>httpcore-nio</artifactId>
+			 <version>4.4.12</version>
+		  </dependency>
+	   </dependencies>
+	</dependencyManagement>
+
 	<build>
 		<plugins>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-shade-plugin</artifactId>
+				<artifactId>maven-jar-plugin</artifactId>
 				<executions>
 					<execution>
-						<id>Elasticsearch6SinkExample</id>
+						<id>Jar Package</id>
 						<phase>package</phase>
 						<goals>
-							<goal>shade</goal>
+							<goal>test-jar</goal>
```

Comment:
Can you avoid introducing more test jars and maybe move the common utilities to the main package?